packages/roosterjs-content-model-dom/lib/modelToDom/handlers/handleBlockGroupChildren.ts

```diff
@@ -39,6 +40,14 @@ export const handleBlockGroupChildren: ContentModelHandler<ContentModelBlockGrou
             }
         });
 
+        if (
+            group.blockGroupType == 'Document' &&
```

Comment:
by the name of the function, handleBlockGroupChildren,  it only should handle only the children of a blockGroup. Should we create handler just to handle the parent?

Reply:
I think we can just move this to contentModelToDom, good catch.